src/data/gen2Banner-urls.ts

```diff
@@ -29,57 +57,196 @@ export const GEN2BANNER_URLS = [
+  '/nextjs/build-a-backend/graphqlapi/subscribe-data/',
   '/javascript/tools/cli/graphqlapi/data-modeling/',
+  '/angular/tools/cli/graphqlapi/data-modeling/',
+  '/react/tools/cli/graphqlapi/data-modeling/',
+  '/nextjs/tools/cli/graphqlapi/data-modeling/',
+  '/vue/tools/cli/graphqlapi/data-modeling/',
   '/javascript/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/react/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/angular/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/vue/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/nextjs/build-a-backend/graphqlapi/customize-authorization-rules/',
   '/javascript/tools/cli/graphqlapi/custom-business-logic/',
+  '/angular/tools/cli/graphqlapi/custom-business-logic/',
+  '/react/tools/cli/graphqlapi/custom-business-logic/',
+  '/nextjs/tools/cli/graphqlapi/custom-business-logic/',
+  '/vue/tools/cli/graphqlapi/custom-business-logic/',
   '/javascript/tools/cli/graphqlapi/override-appsync/',
+  '/angular/tools/cli/graphqlapi/override-appsync/',
+  '/react/tools/cli/graphqlapi/override-appsync/',
+  '/nextjs/tools/cli/graphqlapi/override-appsync/',
+  '/vue/tools/cli/graphqlapi/override-appsync/',
   '/javascript/build-a-backend/graphqlapi/offline/',
+  '/react/build-a-backend/graphqlapi/offline/',
+  '/angular/build-a-backend/graphqlapi/offline/',
+  '/vue/build-a-backend/graphqlapi/offline/',
+  '/nextjs/build-a-backend/graphqlapi/offline/',
   '/javascript/build-a-backend/graphqlapi/best-practice/',
+  '/react/build-a-backend/graphqlapi/best-practice/',
+  '/angular/build-a-backend/graphqlapi/best-practice/',
+  '/vue/build-a-backend/graphqlapi/best-practice/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/',
   '/javascript/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/angular/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/react/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/vue/build-a-backend/graphqlapi/best-practice/deltasync/',
   '/javascript/build-a-backend/graphqlapi/best-practice/create-blog/',
-  '/javascript/build-a-backend/graphqlapi/best-practice/wharehouse-management/',
+  '/angular/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/react/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/vue/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/javascript/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/angular/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/react/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/vue/build-a-backend/graphqlapi/best-practice/warehouse-management/',
   '/javascript/tools/cli/graphqlapi/search-and-result-aggregations/',
```

Comment:
There was a late move of this page from CLI - including JS all of these should be added to:
[platform]/build-a-backend/graphqlapi/search-and-result-aggregations/